week-12/homework.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Homework week 12</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div class="navbar sticky-top bg-dark d-flex justify-content-between">
+      <a class="navbar-brand col-md-3 text-white" href=".">My app</a>
+      <button
+        class="navbar-toggler text-white bi bi-list me-1"
+        type="button"
+        data-bs-toggle="offcanvas"
+        data-bs-target="#offcanvasNavbar"
+        aria-controls="offcanvasNavbar"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon bi bi-list me-1"></span>Menu
+      </button>
+    </div>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-md-9 col-lg-10 p-4">
+          <div id="carouselExample" class="carousel slide">
```

Comment:
Even when we copy the HTML code, we should check and modify it, `id="carouselExample"` is not a great id to use for homework, you should name it something like `id="products-carousel"`